docs/faq.rst

```diff
@@ -10,7 +10,18 @@ FAQ
 [tedana] How do I use tedana with fMRIPrepped data?
 ***************************************************
 
-`fMRIPrep`_ outputs the preprocessed, optimally-combined fMRI data, rather than echo-wise data.
+fMRIPrep versions >= 21.0.0
+===========================
+
+Starting with version 21.0.0, `fMRIPrep`_ added the ``--me-output-echos`` argument,
+which outputs individual echoes after slice timing, motion correction, and distortion correction have been performed.
+These preprocessed echoes can be denoised with tedana,
+after which warps written out by fMRIPrep can be applied to transform the denoised data to standard space.
+
+fMRIPrep versions < 21.0.0
+==========================
+
+Prior to version 21.0.0, `fMRIPrep`_ outputted the preprocessed, optimally-combined fMRI data, rather than echo-wise data.
```

Comment:
Technically, it could also output preprocessed, individual echos using the `--echo-idx` option. It might be worth bringing this up only in that previously, folks might have thought they could use those echos in tedana (which we tried to discourage).